src/main/kotlin/no/nav/familie/ef/sak/opplysninger/mapper/BarnMatcher.kt

```diff
@@ -60,20 +71,16 @@ data class MatchetBehandlingBarn(
 )
 
 fun finnBesteMatchPåFødselsnummerForTermindato(
-    fødselsnumre: List<String>,
+    barn: List<BarnMedFødselsdatoDto>,
     termindato: LocalDate,
 ): String? {
     val uke20 = termindato.minusWeeks(20)
     val uke44 = termindato.plusWeeks(4)
 
-    return fødselsnumre
+    return barn
         .filter {
-            val fødselsnummer = Fødselsnummer(it)
-            val fødselsdato = fødselsnummer.fødselsdato
-            fødselsdato.isBefore(uke44) and fødselsdato.isAfter(uke20)
-        }.minByOrNull {
-            val epochDayForFødsel = Fødselsnummer(it).fødselsdato.toEpochDay()
-            val epochDayTermindato = termindato.toEpochDay()
-            abs(epochDayForFødsel - epochDayTermindato)
-        }
+            it.fødselsdato != null && (it.fødselsdato.isBefore(uke44) and it.fødselsdato.isAfter(uke20))
+        }.minByOrNull { barn ->
```

Comment:
```suggestion
        }.minByOrNull { aktueltBarn ->
```

`barn` er input over her, så blir shadow variabel